.github/workflows/publish.yml

```diff
@@ -1,17 +1,21 @@
 name: Publish to PyPI
 on:
   push:
-    tags:
-      - 'v[1-9][0-9].[0-9]+.[0-9]+'
```

Comment:
I saw that this is no longer included? Does this mean that we can't trigger a dev release manually?

Reply:
Good catch, I've added it back in